packages/patternfly-4/react-core/src/components/Progress/ProgressContainer.js

```diff
@@ -42,14 +42,16 @@ const variantToIcon = {
   [ProgressVariant.success]: CheckCircleIcon
 };
 
-const ProgressContainer = ({ value, title, parentId, variant, measureLocation }) => {
+const ProgressContainer = ({ value, title, parentId, label, variant, measureLocation }) => {
   const StatusIcon = variantToIcon.hasOwnProperty(variant) && variantToIcon[variant];
   return (
     <Fragment>
       <div className={css(styles.progressDescription)} id={`${parentId}-description`}>
         {title}
       </div>
       <div className={css(styles.progressStatus)}>
+        {/* TODO: add padding to label */}
+        {label}
         {(measureLocation === ProgressMeasureLocation.top || measureLocation === ProgressMeasureLocation.outside) && (
           <span className={css(styles.progressMeasure)}>{value}%</span>
         )}
```

Comment:
I'm looking to wrap the percentage in parentheses like the snapshot below. It looks like I can only add text before it?

![screen shot 2018-10-16 at 12 49 51 pm](https://user-images.githubusercontent.com/17481322/47033469-e4b64f80-d142-11e8-9ee0-1d6d4ada19aa.png)

I'd prefer if the label was wrapped in the span so we get the same text style provided by progressMeasure. Perhaps we can use the `value` by default, if `label` has not been provided?

Reply:
Your example fits this as well, you'll just use `measureLocation={ProgressMeasureLocation.none}` and pass your own label which will be whatever you'll want (string, node, number). If we'd were to force parentheses we'll be targeting only this specific example. But what if someone would want to show `Already donwloaded: 50 %`? We'd have to have another branching here.

But you are correct that I should add styles applied to progress number on top of those custom styles (so label has some padding).

Reply:
Not asking to force parentheses. I just want the ability to provide my own label. If measureLocation, will hide it, that's fine. Just need the styling.

Reply:
With this approach you'll be able to pass your own label. I think we don't understand each other on here. Check the [example](https://github.com/patternfly/patternfly-react/blob/f50624a204629638a068e28c441c55990e421e72/packages/patternfly-4/react-core/src/components/Progress/examples/LabeledProgress.js) where you can pass whatever you want
```JSX
<Progress value={33} title="Descriptive text here" label={<div>$106.30 (82.49%)</div>} measureLocation={ProgressMeasureLocation.none} />;
```
Or just plain old string
```JSX
<Progress value={33} title="Descriptive text here" label="$106.30 (82.49%)" measureLocation={ProgressMeasureLocation.none} />;
```

Reply:
@karelhala I think he is more speaking to line 56 here since in addition to the label passed the component is also adding in ` <span className={css(styles.progressMeasure)}>{value}%</span>` that would make this:
```jsx
<Progress value={33} title="Descriptive text here" label="$106.30 (82.49%)"  />;
```
Output something like `$106.30 (82.49%) 33%` from the looks of it. 

Is there a way that we can have them just override the actual measure label? like David mentioned, default to the `{value}%`, but user the one provided if there is one. Someone might want to override the label no matter the possition.